src/emulator/storage/rules/manager.ts

```diff
@@ -157,12 +150,6 @@ class ResourceBasedStorageRulesManager implements StorageRulesManager {
     return this._rulesManagers.get(resource)?.getRuleset();
   }
 
-  updateSourceFile(rules: SourceFile, resource: string): Promise<StorageRulesIssues> {
```

Comment:
I just noticed `ResourceBasedStorageRulesManager` is missing hot reloading of storage rules. Can you add that in too?

Reply:
Sorry you're right, forgot about the relationship between the two